_data/banking.yml

```diff
@@ -663,8 +663,9 @@ websites:
       url: https://seb.se/
       img: seb.png
       tfa: Yes
+      software: Yes
```

Comment:
SEB have a Digipass option for 2FA. It's a Vasco hardware token that do not require Swedish citizenship or residency.